### PR TITLE
update test-dapp to 5.0.0

### PR DESCRIPTION
**Pull request by brad-decker**
Explanation: Update to the latest test-dapp package
- Updated `edit-gas-fee.spec.js` as the Send flow on the test-dapp was updated, reducing the ETH amount to send from 3 to 0  https://github.com/MetaMask/test-dapp/pull/129
- Update `signature-request.spec.js` as the signTypedData_v3 message was fixed in the test-dapp https://github.com/MetaMask/test-dapp/pull/133

### CODE REVIEW

File: test/e2e/tests/edit-gas-fee.spec.js [hunk 1]
```diff
@@ -238,17 +241,17 @@ describe('Editing Confirm Transaction', function () {
           '.currency-display-component__text',
         );
         const transactionAmount = transactionAmounts[0];
-        assert.equal(await transactionAmount.getText(), '3');
+        assert.equal(await transactionAmount.getText(), '0');
```
Comment (adonesky1) [inline test/e2e/tests/edit-gas-fee.spec.js:244]: Did we lower the amount that the send transaction button sends?

Comment (brad-decker) [reply test/e2e/tests/edit-gas-fee.spec.js:244]: we did

>Updated edit-gas-fee.spec.js as the Send flow on the test-dapp was updated, reducing the ETH amount to send from 3 to 0 https://github.com/MetaMask/test-dapp/pull/129

File: test/e2e/tests/signature-request.spec.js [hunk 2]
```diff
@@ -126,7 +126,7 @@ describe('Sign Typed Data V3 Signature Request', function () {
         );
         const origin = content[0];
         const address = content[1];
-        const message = await driver.findElement(
+        const message = await driver.findElements(
```
Comment (mcmire) [inline test/e2e/tests/signature-request.spec.js:129]: Should this be `messages` since there are now more than one?

**Review by mcmire — DISMISSED** [review]
One nit, but otherwise looks good.

File: test/e2e/tests/edit-gas-fee.spec.js [hunk 3]
```diff
@@ -216,7 +216,10 @@ describe('Editing Confirm Transaction', function () {
 
         // open dapp and connect
         await connectDappWithExtensionPopup(driver);
-        await driver.clickElement({ text: 'Send', tag: 'button' });
+        await driver.clickElement({
+          text: 'Send EIP 1559 Transaction',
```
Comment (adonesky1) [inline test/e2e/tests/edit-gas-fee.spec.js:220]: Do we want to also still do a test for `Send Legacy Transaction`?

File: test/e2e/tests/signature-request.spec.js (hunk 2 above)
Comment (PeterYinusa) [reply test/e2e/tests/signature-request.spec.js]: Updated

File: test/e2e/tests/edit-gas-fee.spec.js (hunk 3 above)
Comment (PeterYinusa) [reply test/e2e/tests/edit-gas-fee.spec.js:220]: Yes, that test lives in send-eth.spec.js

Comment (adonesky1) [reply test/e2e/tests/edit-gas-fee.spec.js:220]: Ah ok!

**Review by mcmire — APPROVED** [review]

**Review by adonesky1 — APPROVED** [review]

**Pull request merged**